test/integration-test/Diagnostics/err_gsl_uninitialized_temporary.cpp

```diff
@@ -22,7 +22,7 @@ using namespace gridtools::clang;
 
```

Comment:
Can you remove the gsl from the filename -- probably some form of artifact from a long time ago ;)

Reply:
:)